### ISSUE TITLE
Możliwość pracy z kilkoma plikami konfiguracyjnymi

### ISSUE THREAD

**Issue by bzykbzyk** [uzi18/sterownik#23]
Można by zdefiniować kilka plików np z parametrami dla różnych typów paliwa i czytać w razie potrzeby.

**Comment by Stan17** [comment]
Tak, o taką możliwość wielokrotnie proszono ES dla ich sterowników, sam również o to prosiłem.

**Comment by uzi18** [comment]
mhm, czyli raczej opcje spalania w oddzielnym pliku ..., a jak sobie wyobrazacie wybor tych plikow?

**Comment by bzykbzyk** [comment]
Moim zdaniem wszystko powinno być w jednym pliku. Program przy uruchamianiu czyta parametry z pliku config.txt. Na stronie www mamy możliwość wyboru pliku przez najzwyklejsze otwórz plik lub zapis bieżącej konfiguracji z pamięci pod dowolną nazwą w celu łatwego orientowania się co tam mamy. Podczas zapisu program powinien zapisać ją pod nową nazwą i jednocześnie zastąpić config.txt . Gwarantowało by to po ponownym uruchomieniu odczyt ostatniej konfig bez kombinacji z pamiętaniem przez program, co było ostatnio. Jak coś pochrzanimy w konf, wystarczy na karcie zapisać poprawny plik config.txt i wszystko ok.

**Comment by uzi18** [comment]
zobacze co da sie zrobic, plikami bedziemy zarzadzac poprzez www, to pewne.
moze czesc parametrow zachowam w eeprom - te kluczowe np. ip,mac,maska sieci i hasla do strony.
wtedy w konfigu beda one (te dane) potrzebne tylko w wypadku pierwszego uruchomienia

**Comment by bzykbzyk** [comment]
No właśnie, jak działa ten tajny program uzi18'ego. Kiedy czyta config np.

**Comment by uzi18** [comment]
konfig jest czytany zaraz po resecie

**Comment by bzykbzyk** [comment]
Czyli również po uruchomieniu?

**Comment by uzi18** [comment]
zgadza się

**Comment by Stan17** [comment]
Podbijam temat, przydała by się taka możliwość do testów steownika.

**Comment by uzi18** [comment]
Mozesz miec kilka konfiguracji na komputerze i wysylac raz ta raz tamta.
Dla ulatwienia skopiowac sobie i zmodyfikowac skrypt do wysylania configu
aby konkretny plik wysylal.
